### PR TITLE
feat(animation): add motion tokens

### LINKED ISSUE #52: Tokens - import motion tokens
Import motion tokens from Figma.

### PR DESCRIPTION
**Pull request by srambach**
Fixes #52 

@mcoker @lboehling @kaylachumley Do you think the base tokens (like --pf-t--global--delay--400) should have "motion" in them?

@mcoker Is it ok to include these in the -default scss file?

### CODE REVIEW

**Comment by mcoker** [comment]
> @mcoker @lboehling @kaylachumley Do you think the base tokens (like --pf-t--global--delay--400) should have "motion" in them?

@srambach I'm curious if others have an opinion. A little bit on what I'm thinking is...

Pros to not including `--motion`:
* Vars are shorter, which isn't terribly important but it's more than zero importance!
* Naming is more flexible so they could be used with other  non-motion semantic tokens without the naming being potentially confusing. This implies they may be used for things other than `--motion` tokens

Pros to including `--motion`:
* being more clear as to what these apply to. Everything is under `--pf-t--global--motion`. Implies these base tokens will probably only apply to `--pf-t--global--motion` semantic vars
* avoiding namespace conflicts if we introduced different base delays/durations for some other non-motion tokens

> @mcoker Is it ok to include these in the -default scss file?

I don't see why not. Is there a potential issue you can think of?

File: packages/module/build.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const build = (selector) => {
     name: 'patternfly/global/px',
     type: 'value',
     matcher: (token) =>
+      token.attributes.type === 'breakpoint' ||
```
Comment (mcoker) [inline packages/module/build.js:29]: Just wanted to make sure this change was intentional 👍

**Review by mcoker — APPROVED** [review]
lgtm! just one comment about an update to build.js

**Comment by srambach** [comment]
Closing in favor of #56, which brings in the relevant tokens.
As to naming... let's go with this for the moment.
And motion tokens are included in the default scss token file.

**Pull request closed without merging**